REFERENCE.md

```diff
@@ -0,0 +1,363 @@
+    {
+        try {
+            Init init;
+            String consumerSecret = "74c5fd430cf1242a527f6223aebd42d30464be22";
+
+            String secString = "{\"consumer_key\":\"yis0TYCu7U9V4o7M\","
+                    +    "\"domain\": \"assess.vg.learnosity.com\","
+                    +    "\"user_id\": \"$ANONYMIZED_USER_ID\"}";
+
+            String reqString = "{\"state\":\"initial\","
+                    +  "\"type\":\"local_practice\","
+                    +  "\"response_id\":\"60005\","
+                    +  "\"questions\":"
+                    +   "[{\"stimulus_list\":"
+                    +     "[\"London\","
+                    +        "\"Dublin\","
+                    +        "\"Paris\","
+                    +      "\"Sydney\"],"
+                    + "\"stimulus\":\"Match the cities to the parent nation\","
+                    +   "\"type\":\"association\","
+                    +   "\"possible_responses\":"
+                    +   "[\"Australia\","
+                    +     "\"France\","
+                    +     "\"Ireland\","
+                    +       "\"England\"],"
+                    +   "\"validation\":"
+                    +   "{\"valid_responses\":"
+                    +       "[\"England\","
+                    +             "\"Ireland\","
+                    +       "\"France\","
+                    +       "\"Australia\"]}}]}";
+            init = new Init("questions", secString, consumerSecret, new JSONObject(reqString));
+            String questionJson = init.generate();
+            System.out.println(questionJson);
+
+            // Now you can pass questionJson to the initialisation of Questions API
+            // For instance on jsp you might have
+            // <script src="//questions.learnosity.com"></script>
+            // var questionsApp = LearnosityApp.init(questionJson);
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+        }
+    }
+}
+```
+
+#### Arguments
+**service**<br>
+A string representing the Learnosity service (API) you want to integrate with. Valid options are:
+
+* assess
+* author
+* data
+* items
+* questions
+* reports
+
+**security**<br>
+An Object that includes your *consumer_key* but does not include your *secret*. The SDK sets defaults for you, but valid options are:
+
+* consumer_key
+* domain (optional)
+* timestamp (optional)
+* user_id (optional)
+
+^Note – the SDK accepts all Objects which can be parsed into a org.json.JSONObject (e.g. a map, a JSONString, JSONOBject or a Java bean).
+
+**secret**<br>
+Your secret key, as provided by Learnosity.
+
+**request**<br>
+An optional associative Object of data relevant to the API being used. This will be any data minus the security details that you would normally use to initialise an API.
+
+^Note – the SDK accepts all Objects which can be parsed into a org.json.JSONObject (e.g. a map, a JSONString, JSONOBject or a Java bean).
+
+
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 2](https://app.codacy.com/gh/Learnosity/learnosity-sdk-java/pullRequest?prid=8556170)
```suggestion

```